reconcile/utils/terrascript_client.py

```diff
@@ -3334,6 +3340,25 @@ def populate_tf_resource_elasticsearch(self, resource, namespace_info):
         for tf_resource in tf_resources:
             self.add_resource(account, tf_resource)
 
+    def _build_es_advanced_security_options(
+            self, advanced_security_options: dict[str, Any]) -> dict[str, Any]:
```

Comment:
According to [dev-guidelines](https://service.pages.redhat.com/dev-guidelines/docs/appsre/python-style-guide/#abstract-collection-types), my understanding here would be:

```
def _build_es_advanced_security_options(
            self, advanced_security_options: Mapping[str, Any]) -> Mapping[str, Any]:
```

Reply:
It actually needs to be a mutable mapping but you're right. The method signature helps understand better the code now

Fixed in 3186d1e